src/Api/Objects.php

```diff
@@ -123,6 +123,20 @@ public function bulkDelete(string $collection, array $objectIds, array $headers
         return $this->postRequest($url, ['object_ids' => $objectIds], $headers);
     }
 
+    /**
+     * @param string $collection
+     * @param array $subscriptions
+     * @param array $headers
+     * @return array
+     * @throws Exception
+     */
+    public function bulkAddSubscriptions(string $collection, array $subscriptions, array $headers = []): array
```

Comment:
Does it really return an array? Asking because in other SDK updates, we would return a bulk operation? [example](https://github.com/knocklabs/knock-ruby/pull/15/files#diff-8f7de4cec347c11bc93489b7a4c53067d501c63732598a6ff259cb750c96d0b7R372)

Reply:
Yea. `array` here just means a JSON blob. None of the other functions in this SDK type the API data response (every one is `@return array`).